core/arch/opnd.h

```diff
@@ -2418,13 +2418,20 @@ reg_32_to_opsz(reg_id_t reg, opnd_size_t sz);
 
 DR_API
 /**
- * Given a general-purpose register of any size, returns a register in the same
- * class of the given size.  For example, given \p DR_REG_AX or \p DR_REG_RAX
- * and \p OPSZ_1, this routine will return \p DR_REG_AL.
+ * Given a general-purpose or SIMD register of any size, returns a register in the same
+ * class of the given size.
+ *
+ * For example, given \p DR_REG_AX or \p DR_REG_RAX and \p OPSZ_1, this routine will
+ * return \p DR_REG_AL. Given \p DR_REG_XMM0 and \p OPSZ-64, it will return \p
+ * DR_REG_ZMM0.
+ *
  * Returns \p DR_REG_NULL when trying to get the 8-bit subregister of \p
  * DR_REG_ESI, \p DR_REG_EDI, \p DR_REG_EBP, or \p DR_REG_ESP in 32-bit mode.
  * For 64-bit versions of this library, if \p sz == OPSZ_8, returns the 64-bit
  * version of \p reg.
+ *
+ * MMX registers are not yet supported.
+ * Moreover, ARM is not yet supported.
```

Comment:
Sorry now it sounds like ARM is not supported at all. Could you spell it out in a way that makes it clear that only ARM SIMD is not supported in the context of this function... Thanks

Reply:
Done